ceph.spec.in

```diff
@@ -403,7 +402,6 @@ Summary:	OCF-compliant resource agents for Ceph daemons
 %if 0%{?suse_version}
 Group:		System/Filesystems
 %endif
-License:	LGPL-2.0
```

Comment:
>  i am just not sure why we have separated licenses for the subpackages in the very beginning. did we add them because it's "necessary"? if not, it'd be great to use a single license.

I did some digging, and in the beginning (2010, 2011) the main package had the line "License: LGPLv2" (later changed to "LGPL-2.0"). Later, as the individual subpackages were added, this line was always copy-pasted. I doubt anyone put much thought into it.